ios/RNCProgressView.h

```diff
@@ -1,11 +0,0 @@
-
```

Comment:
Could you keep the `RNCProgressView.h` (with capital V) and delete `RNCProgressview.h` instead? :)

Reply:
huh, makes sense, I didn't event notice that :D will do